### PR TITLE
Update missinglink-core to 0.1.5

### PR DESCRIPTION
**Pull request by LukaJCB**

### CODE REVIEW

**Comment by LukaJCB** [comment]
I'm not sure why CI failed this, a cache problem maybe? It doesn't seem to pull in 0.1.5

**Comment by iRevive** [comment]
@LukaJCB  
I've updated missinglink version in the `<root>/project/plugins.sbt` and it solved a compilation issue.

Can you uncomment line 105 in the file `MissingLinkPlugin.scala`? 
```
Java9ModuleLoader.getJava9ModuleArtifacts((s, ex) => log.warn(s))
```

Then the plugin will support JDK 9+.

**Comment by sjrd** [comment]
@LukaJCB To fix the CI you also need to change the version number in https://github.com/scalacenter/sbt-missinglink/blob/master/project/plugins.sbt

**Comment by LukaJCB** [comment]
Thanks!

File: src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala [hunk 1]
```diff
@@ -96,7 +95,7 @@ object MissingLinkPlugin extends AutoPlugin {
   }
 
   private def loadClass(f: File): DeclaredClass = {
-    new com.spotify.missinglink.ClassLoader(new FileInputStream(f)).load()
+    com.spotify.missinglink.ClassLoader.load(new FileInputStream(f))
```
Comment (plokhotnyuk) [inline src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala]: How to make sure that there no leak of file descriptors here?

**Pull request merged**